src/scripts/battleFrontier/BattleFrontierRunner.ts

```diff
@@ -67,10 +67,10 @@ class BattleFrontierRunner {
 
         Notifier.notify({
             title: 'Battle Frontier',
-            message: `You managed to beat stage ${stageBeaten}.\nYou received ${battlePointsEarned} BP`,
+            message: `You managed to beat stage ${stageBeaten}.\nYou received ${battlePointsEarned} BP.\nYou received ${moneyEarned}.`,
```

Comment:
There should also be something to identify that the 2nd amount is money,
Any one of these should do:
```
<img src='moneyimagelocation'/> xxx
xxx money (or pokedollars, don't remember what we call it 😅)
$xxx
```
Could you also `toLocaleString` the amounts (should probably also be done to the BP amount).